src/quacc/calculators/espresso/espresso.py

```diff
@@ -28,7 +28,7 @@ class EspressoTemplate(EspressoTemplate_):
     of other binaries such as pw.x, ph.x, cp.x, etc.
     """
 
-    def __init__(self, binary: str = "pw") -> None:
+    def __init__(self, binary: str = "pw", test_run: bool = False) -> None:
```

Comment:
When you're done, make sure to add the docstring :) 

Note: There's a small typo below ("we fall bacl on" --> "we fall back to")